packages/i18nify-js/src/modules/phoneNumber/data/phoneFormatterMapper.json

```diff
@@ -249,5 +242,21 @@
   "ZM": "xxx-xxxxxxx",
   "ZW": "xx xxx xxxx",
   "XK": "xxx xxx xxx",
-  "FW": "xxx xx xx xx"
-}
\ No newline at end of file
+  "FW": "xxx xx xx xx",
+  "YT-UNF": "xxx xx xx xx",
+  "UM": "",
```

Comment:
What happens in case of empty string formats ?

Reply:
Below are all the utilities that use this  and their behaviours in such cases.

For such cases: 


1. getMaskedContactNumber
   -  throws error when country code is UM and no phone number is passed: `i18nify Error: Error: Parameter "countryCode" is invalid: UM`
   - In case country code is UM and phone number is passed: it performs complete masking of entire number without formatting eg: 7394926646 --> xxxxxxxxxx

2. formatPhoneNumber returns the inputted phone number as it is.

3. parsePhoneNumber
    - For phoneNumber = '+917394926646', country = 'UM' output is
       ```
       {
          countryCode: 'UM',
          dialCode: '+91',
          formattedPhoneNumber: '+917394926646',
          formatTemplate: '',
          phoneNumber: '+917394926646'
        }
       ```
    - For phoneNumber = '7394926646', country = 'UM' output is
       ```
       {
        countryCode: 'UM',
        dialCode: '',
        formattedPhoneNumber: '7394926646',
        formatTemplate: '',
        phoneNumber: '7394926646'
      }
       ```